src/utils/utils.ts

```diff
@@ -27,3 +16,48 @@ export function getNextSortValue(value?: "asc" | "desc" | null) {
       return "asc";
   }
 }
+
+export function filterData<T>(
+  data: T[],
+  column: Filter,
+  columnsArr: Column[]
+): T[] {
+  if (isEmpty(column)) return data;
+  return data.filter((row) => {
+    return Object.entries(column).every(([columnName, filterInputValue]) => {
+      const exactMatchValue = columnsArr.find(
```

Comment:
We tend to call 'booleanish' vars with `is{...}` prefix - `const isExactMatchValue = ...`, bc then whoever reads the code understands quickly that it's used as a flag check